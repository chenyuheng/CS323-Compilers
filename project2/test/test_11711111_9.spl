struct node {
    int a, b, c;
};

struct node{
    float a, b, c;
};
