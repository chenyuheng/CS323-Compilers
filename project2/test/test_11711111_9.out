Error type 15 at Line 5: redefine the same structure type
